mk/config.mk

```diff
@@ -176,6 +176,31 @@ CFG_WITH_USER_TA ?= y
 # case you implement your own TA store
 CFG_REE_FS_TA ?= y
 
+# Support for loading user TAs from a special section in the TEE binary.
+# Such TAs are available even before tee-supplicant is available (hence their
+# name), but note that many services exported to TAs may need tee-supplicant,
+# so early use is limited to a subset of the TEE Internal Core API (crypto...)
+# To use this feature, set EARLY_TA_PATHS to the paths to one or more TA ELF
+# file(s). For example:
+#   $ make ... \
+#     EARLY_TA_PATHS=path/to/8aaaf200-2450-11e4-abe2-0002a5d5c51b.stripped.elf
+# or:
+#   $ make ... \
+#     EARLY_TA_PATHS="path/to/8aaaf200-2450-11e4-abe2-0002a5d5c51b.stripped.elf \
+#                     path/to/cb3e5ba0-adf1-11e0-998b-0002a5d5c51b.stripped.elf"
+# Typical build steps:
+#   $ make ... CFG_EARLY_TA=y # Enable the feature to avoid later recompilation
```

Comment:
As CFG_EARLY_TA=y,  EARLY_TA_PATHS=test_ta_uuid.
When test_ta and optee_os are both modified, need to make twice optee_os. 
May it add "make headers_install"?  Build headers for TA first, then build TA, finally build optee_os.